app/src/main/java/woowacourse/shopping/view/cart/CartItemViewHolder.kt

```diff
@@ -0,0 +1,45 @@
+package woowacourse.shopping.view.cart
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import woowacourse.shopping.R
+import woowacourse.shopping.databinding.ItemCartBinding
+import woowacourse.shopping.databinding.ItemCartPaginationBinding
+import woowacourse.shopping.model.ProductModel
+import woowacourse.shopping.util.PriceFormatter
+
+sealed class CartItemViewHolder(view: View): RecyclerView.ViewHolder(view) {
+    class CartProductViewHolder(private val binding: ItemCartBinding) : CartItemViewHolder(binding.root) {
+        fun bind(product: ProductModel, onItemClick: CartAdapter.OnItemClick) {
+            binding.cartProduct = product
+            binding.textPrice.text = binding.root.context.getString(R.string.korean_won, PriceFormatter.format(product.price))
+            Glide.with(binding.root.context).load(product.imageUrl).into(binding.imgProduct)
+            binding.onItemClick = onItemClick
+        }
+    }
+
+    class CartPaginationViewHolder(private val binding: ItemCartPaginationBinding, onItemClick: CartAdapter.OnItemClick) : CartItemViewHolder(binding.root) {
```

Comment:
```suggestion
    class CartPaginationViewHolder(
        private val binding: ItemCartPaginationBinding, 
        onItemClick: CartAdapter.OnItemClick
    ) : CartItemViewHolder(binding.root) {
```
코드가 가로로 길어지는 경우에는, 코드를 가로로 스크롤해서 보아야하기 때문에 불편한 점이 있습니다.
그렇기에 개행을 해주시면 코드를 가로로 스크롤하지 않고 볼 수 있습니다. (이하 관련 내용 동일)